tests/driver_at86rf2xx/common.h

```diff
@@ -41,6 +41,7 @@ extern at86rf2xx_t devs[AT86RF2XX_NUM];
 void recv(netdev_t *dev);
 int ifconfig(int argc, char **argv);
 int txtsnd(int argc, char **argv);
+int random(int argc, char **argv);
```

Comment:
There already is a `random()` function in the standard C lib causing the compilation error. So, this function has to be renamed to get the CI passing :-(

Please squash again after renaming it. I have my fingers crossed that the CI is pleased afterwards and it can be finally be merged :)